harmoni_detectors/harmoni_stt/config/configuration.yaml

```diff
@@ -1,5 +1,13 @@
-harmoni_stt:
+w2l:
     default_param:
         model_path: /root/model/w2l/
         w2l_bin: /root/wav2letter/build/inference/inference/examples/simple_streaming_asr_example
-        subscriber_id: "default"
\ No newline at end of file
+        subscriber_id: "default"
+
+google:
+    default_param:
+        language_id: "it-IT"
```

Comment:
I'm obviously a little biased, but I think our default language should be English. There should probably be a single source of info for all localization config so it's easy to switch. Maybe this is it? Probably don't need to change this right now, but worth thinking about.

Reply:
We do need to put more thought into our configs and how we handle making it easy to switch and know what needs to change.